amo/tx/tx.go

```diff
@@ -19,12 +19,13 @@ const (
 )
 
 var (
-	c = elliptic.P256()
+	// config values from the app
+	ConfigLockupPeriod = defaultLockupPeriod
```

Comment:
This will raise conflict when merging. merge from the recent master branch to avoid conflict.